CHANGELOG.md

```diff
@@ -1,5 +1,81 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.13
+
+<!-- generated comparing v5.15.12..master -->
+
+_Mar 12, 2024_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highights ✨
+
+- The Checkout template has been refined (#41447) @zanivan
+
+### `@mui/material@5.15.13`
+
+- &#8203;<!-- 43 -->[Alert] Add `slots` and `slotProps` type to theme (#41324) @sai6855
+- &#8203;<!-- 41 -->[Autocomplete] Fix the options list being added to the DOM in `freeSolo` mode even when there are no options, causing style problems (#41300) @rakeshmusturi
+- &#8203;<!-- 15 --> Add `paperChannel` token (#41447) @siriwatknp
+- &#8203;<!-- 13 -->[Switch] Convert to support CSS extraction (#41367) @alexfauquette
+
+### `@mui/system@5.15.13`
+
+- &#8203;<!-- 03 -->[RtlProvider] Add component & hook (#41241) @mnajdova
+
+### `@mui/codemod@5.15.13`
+
+- &#8203;<!-- 12 -->[mui-codemod] Fix merging of slotProps and componentProps (#41323) @sai6855
+
+### `@mui/base@5.0.0-beta.39`
+
+- &#8203;<!-- 42 -->[material-ui][joy-ui][Autocomplete] Keep in sync highlighted index when the option still exists (#41306) @CGNonofr
```

Comment:
```suggestion
- &#8203;<!-- 42 -->[Autocomplete] Keep in sync highlighted index when the option still exists (#41306) @CGNonofr
```

Removing the Material and Joy tags? I see that the root change was the primitive change, and then other products just consumed it.

Reply:
I think we should keep them as the change affects these packages.